engine/xml-config/src/main/resources/configuration.xsd

```diff
@@ -140,6 +171,27 @@
 		</sequence>
 	</complexType>
 
+    <complexType name="remoteComponentsType">
+        <sequence>
+            <element name="server" type="ab:serverType"/>
+        </sequence>
+    </complexType>
+
+    <complexType name="serverType">
```

Comment:
Can we qualify this a bit more since this file contains many types of servers (database/datastore servers etc.). How about "removeComponentServer"?